docs/api_static/xrtpy.response.xrt_teem.rst

```diff
@@ -0,0 +1,8 @@
+:orphan:
+
+`xrtpy.response.xrt_teem`
+=========================
+
+.. currentmodule:: xrtpy.response.xrt_teem
+
+.. automodapi:: xrtpy.response.xrt_teem
```

Comment:
When we add a new module, we often need to [add a stub documentation file](Using_xrt_teem) so that the docstrings from the module show up in the online documentation.